flashcard.koplugin/main.lua

```diff
@@ -8,6 +8,10 @@ local _ = require("gettext")
 local T = require("ffi/util").template
 local util = require("ffi/util")
 local DataStorage = require("datastorage")
+local json = require("json")
+local logger = require("logger")
+
+debug = true
 
 CardChoice = {
```

Comment:
Same, should be local.

Reply:
really thought I deleted it

Reply:
I think there is a bug in github or you where looking at older files, it doesn't exist in the current version

Reply:
I probably incorrectly assumed you had provided a polished series of commits. ^_^

Reply:
No, this is definitely here. Did you perhaps forget to push your latest changes?

Reply:
![image](https://user-images.githubusercontent.com/37214951/151607721-a1726305-3d34-4c79-9093-03b58165c840.png)
Not for me ¯\_(ツ)_/¯ really weird

Reply:
You're showing me the same thing I'm seeing? I'm saying it should be:
```suggestion
local CardChoice = {
```

Reply:
God fucking dem it, 
My mind was locked on the debug lol
![image](https://user-images.githubusercontent.com/37214951/151637167-4d963b2b-b3c9-4d32-9381-e603d3ceea44.png)
I just didnt even look at CardChoice, my bad